regression/real.ok

```diff
@@ -14410,6 +14945,589 @@ nan
+(~0.123, 0.123E4, 0.2718281828459045E1): ~0.14857171817154097E3	~0.14857171817154094E3
+(~0.123, 0.123E4, ~0.3141592653589793E1): ~0.1544315926535898E3	~0.15443159265358977E3
+(~0.123, 0.123E4, ~0.123E1): ~0.15252E3	~0.15251999999999998E3
+(~0.123, 0.123E4, ~0.123): ~0.151413E3	~0.15141299999999998E3
+(~0.123, 0.123E2, 0.123E1): ~0.2829000000000001	~0.28290000000000015
+(~0.123, 0.123E2, ~0.123E~2): ~0.151413E1	~0.15141300000000002E1
+(~0.123, 0.2718281828459045E1, 0.123): ~0.21134866490046253	~0.21134866490046256
+(~0.123, 0.123E1, 0.123): ~0.28289999999999996E~1	~0.2829000000000001E~1
+(~0.123, 0.123E1, ~0.123): ~0.27429	~0.27429000000000003
+(~0.123, 0.123E1, ~0.123E~2): ~0.15252	~0.15252000000000002
+(~0.123, 0.11125369292536007E~307, 0.5E~323): ~0.136842042298192E~308	~0.1368420422981926E~308
+(~0.123, 0.11125369292536007E~307, ~0.5E~323): ~0.136842042298193E~308	~0.1368420422981936E~308
+(~0.123, ~0.123E4, 0.3141592653589793E1): 0.1544315926535898E3	0.15443159265358977E3
+(~0.123, ~0.123E4, 0.123E1): 0.15252E3	0.15251999999999998E3
+(~0.123, ~0.123E4, 0.123): 0.151413E3	0.15141299999999998E3
+(~0.123, ~0.123E4, ~0.123E2): 0.13899E3	0.13898999999999998E3
+(~0.123, ~0.123E4, ~0.2718281828459045E1): 0.14857171817154097E3	0.14857171817154094E3
+(~0.123, ~0.123E2, 0.123E~2): 0.151413E1	0.15141300000000002E1
+(~0.123, ~0.123E2, ~0.123E1): 0.2829000000000001	0.28290000000000015
+(~0.123, ~0.2718281828459045E1, ~0.123): 0.21134866490046253	0.21134866490046256
+(~0.123, ~0.123E1, 0.123): 0.27429	0.27429000000000003
+(~0.123, ~0.123E1, 0.123E~2): 0.15252	0.15252000000000002
+(~0.123, ~0.123E1, ~0.123): 0.28289999999999996E~1	0.2829000000000001E~1
+(~0.123, ~0.11125369292536007E~307, 0.5E~323): 0.136842042298193E~308	0.1368420422981936E~308
+(~0.123, ~0.11125369292536007E~307, ~0.5E~323): 0.136842042298192E~308	0.1368420422981926E~308
+(~0.123E~2, 0.123E4, 0.123E1): ~0.2829	~0.28289999999999993
+(~0.123E~2, 0.123E4, 0.123E~2): ~0.151167E1	~0.15116699999999998E1
+(~0.123E~2, 0.123E4, ~0.123E1): ~0.27429E1	~0.27428999999999997E1
+(~0.123E~2, 0.123E2, ~0.123E~2): ~0.16359000000000002E~1	~0.16359E~1
+(~0.123E~2, 0.3141592653589793E1, 0.123E~2): ~0.26341589639154455E~2	~0.2634158963915446E~2
+(~0.123E~2, ~0.123E4, 0.123E1): 0.27429E1	0.27428999999999997E1
+(~0.123E~2, ~0.123E4, ~0.123E1): 0.2829	0.28289999999999993
+(~0.123E~2, ~0.123E4, ~0.123E~2): 0.151167E1	0.15116699999999998E1
+(~0.123E~2, ~0.123E2, 0.123E~2): 0.16359000000000002E~1	0.16359E~1
+(~0.123E~2, ~0.3141592653589793E1, ~0.123E~2): 0.26341589639154455E~2	0.2634158963915446E~2
+(~0.11125369292536007E~307, 0.123, 0.5E~323): ~0.136842042298192E~308	~0.1368420422981926E~308
+(~0.11125369292536007E~307, 0.123, ~0.5E~323): ~0.136842042298193E~308	~0.1368420422981936E~308
+(~0.11125369292536007E~307, ~0.123, 0.5E~323): 0.136842042298193E~308	0.1368420422981936E~308
+(~0.11125369292536007E~307, ~0.123, ~0.5E~323): 0.136842042298192E~308	0.1368420422981926E~308
```

Comment:
By convention, the `real.ok` file should be the output for `x86-linux`; meanwhile, the other `real.$(ARCH)-$(OS).ok` files should be updated.  See https://travis-ci.org/MLton/mlton/jobs/389943124, where there is a large reported diff (although, the `./bin/regression` script does not consider a diff with the `real.sml` test as a failure).

Probably, simplest to remove all of the `real.*.ok` files, generate the `real.amd64-linux.ok` and `real.amd64-darwin.ok` outputs, and replace the contents of `real.ok` with `real.$arch-$os.ok` missing; see, for instance, the `size3.ok` file and `size3.*.ok` files.